src/main/java/net/fabricmc/tinyremapper/ClassInstance.java

```diff
@@ -652,6 +705,7 @@ public static String getMrjName(String clsName, int mrjVersion) {
 	private String name;
 	private int mrjVersion;
 	private String superName;
+	private String signature;
```

Comment:
What is the difference between signature and name?

Reply:
The name is the internal name of the class, signature is the generic signature of the class (which may be `null` if the class isn't parametrised)